boot/freeldr/freeldr/lib/mm/meminit.c

```diff
@@ -288,6 +290,7 @@ MmCheckFreeldrImageFile(VOID)
         (OptionalHeader->SizeOfImage > MAX_FREELDR_PE_SIZE) ||
         (OptionalHeader->SectionAlignment != OptionalHeader->FileAlignment))
     {
+    #ifndef UEFIBOOT
```

Comment:
Same question here: What parts do not hold anymore in UEFI builds?

Reply:
OptionalHeader->Subsystem mostly,
i guess size of image too.

Reply:
OK then I think that if those if-checks don't apply to UEFI, you can surround them completely within ifndef UEFIBOOT, and not just their internal code body.